cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-tool-with-invalid-missing-optional.txt

```diff
@@ -2,10 +2,9 @@
 Executing Build Tools
 ERROR [Ballerina.toml:(10:1,10:20)] missing required field 'mode'
 ERROR [Ballerina.toml:(10:17,10:20)] incompatible type for key 'limit': expected 'INTEGER', found 'STRING'
-WARNING: Skipping execution of build tool openapi(generate-delivery-client) as Ballerina.toml contains errors
-
+WARNING: Skipping execution of build tool openapi(generate-delivery-client) contains errors
```

Comment:
We can improve the warning.

E.g. `WARNING: Execution of the build tool 'openapi' for 'generate-delivery-client' is skipped due to errors`

Reply:
Fixed with 64f55075148e69e3b5992080ba8c3b817d5cbc61